src/SortBy/SortBy.jsx

```diff
@@ -1,16 +1,31 @@
 import classNames from 'classnames/bind';
 import PropTypes from 'prop-types';
 import React from 'react';
+import { connect } from 'react-redux';
 
 import styles from './SortBy.scss';
+import { setSortBy } from '../actions';
 
-const SortBy = ({ movieDetails }) => {
+const SortBy = ({ movieDetails, sortBy, dispatch }) => {
   const cx = classNames.bind(styles);
+  const handleSortBy = (value) => {
+    dispatch(setSortBy(value));
+  };
   return !(movieDetails) ? (
```

Comment:
I guess we need to take films from the redux store instead of movieDetails. And if they exist then we show sortBy component